contracts/src/utils/pausable.rs

```diff
@@ -204,6 +206,10 @@ mod tests {
         let res = contract.pause();
         assert!(res.is_ok());
         assert_eq!(contract.paused(), true);
+        let expected_event = Paused { account: msg::sender() };
+        assert!(motsu::emits_event(expected_event.clone()));
```

Comment:
How about changing logic for `motsu::emits_event`, that it would take an event from collection in case of match. No need in calling `motsu::clear_events()` after.